src/engine/srv.c

```diff
@@ -1261,18 +1260,20 @@ dss_sys_db_init()
 		return rc;
 	}
 
-	if (lmm_db_path == NULL) {
-		D_ERROR("DAOS_LMM_DB_PATH need be configured\n");
-		return -DER_INVAL;
+	rc = bio_get_lmm_db_path(lmm_db_path);
```

Comment:
Instead of calling a BIO exported API, I think we could just parse the 'dss_nvme_conf'.